<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerInfoWindow</class>
 <widget class="QWidget" name="PlayerInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="PlayerFieldsGroupBox">
     <property name="title">
      <string>Main info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="PlayerFieldsVerticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3"/>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="PlayerField_1">
            <property name="FieldTag" stdset="0">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="PlayerField_2">
            <property name="FieldTag" stdset="0">
             <string>Age</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="PlayerField_3">
            <property name="FieldTag" stdset="0">
             <string>Work</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="PlayerField_4">
            <property name="FieldTag" stdset="0">
             <string>Last field</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="UpdatePlayerInfoFields">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
